app/views/components/_form_user.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container">
+  <div class="row d-flex justify-content-center">
+    <div class="col-xs-12 col-sm-12 col-md-10 col-lg-8">
+      <div class="create-project">
+          <h1>Editing: <%= @user.first_name %>'s Profile</h1>
+        <%= simple_form_for @user do |f| %>
+          <%= f.input :first_name, placeholder: @user.first_name %>
+          <%= f.input :last_name, placeholder: @user.first_name %>
```

Comment:
@user.last_name maybe? no biggie if so, I'll change it

Reply:
Hey yeah, I figured it will be clicking from your profile/dashboard page + the last name is in the f.input placeholder.